zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ExportingEndpointIT.java

```diff
@@ -106,6 +106,35 @@ void shouldResumeExporting() {
     Awaitility.await().untilAsserted(this::allPartitionsExporting);
   }
 
+  @Test
+  void shouldStayPausedAfterRestart() {
+    // given
+    // in case this test gets run right after shouldPauseExporting
+    getActuator().resume();
+    client
+        .newPublishMessageCommand()
+        .messageName("Test")
+        .correlationKey("5")
+        .messageId("3")
+        .send()
+        .join();
+
+    final var recordsBeforePause =
+        Awaitility.await()
+            .atMost(Duration.ofSeconds(30))
+            .during(Duration.ofSeconds(5))
+            .until(RecordingExporter.getRecords()::size, hasStableValue());
+
+    // when
+    getActuator().pause();
+    CLUSTER.shutdown();
+    CLUSTER.start();
+
+    // then
+    RecordingExporter.records().limit(recordsBeforePause + 1).await();
```

Comment:
:x: If you are not writing any new records, it won't export anyway. So to complete the test, we have to write new records and verify that it is not exported.